src/main/resources/templates/fragments/_bookBrowser.html

```diff
@@ -17,16 +17,16 @@ <h2>Book Catalogue</h2>
             </tr>
             </thead>
             <tbody>
-            <tr th:each="book : ${books}">
```

Comment:
done in a separate pr if we really want to change this

Reply:
The change was moved to the book controller to minimize the number of changes